backend/src/graphql/resolver/TransactionLinkResolver.test.ts

```diff
@@ -23,6 +23,11 @@ import { User } from '@entity/User'
 import { UnconfirmedContribution } from '@model/UnconfirmedContribution'
 import Decimal from 'decimal.js-light'
 import { GraphQLError } from 'graphql'
+import { TRANSACTIONS_LOCK } from '@/util/TRANSACTIONS_LOCK'
+
+// mock semaphore to allow use fake timers
+jest.mock('@/util/TRANSACTIONS_LOCK')
+TRANSACTIONS_LOCK.acquire = jest.fn().mockResolvedValue(jest.fn())
```

Comment:
Why is the TRANSACTION_LOCK mocked here, when it is never used in this file?
Or i can't see or understand the usage of it in line 193 with the statement:

setTimeout(jest.fn(), 1000 * 60 * 60 * 24)

Reply:
The problem is that [here](https://github.com/gradido/gradido/pull/2458/files/7023fdba2a43826c2d8a4c9a2642ab64b38acde4#diff-d177c02b13a87d9ee1ffa3a6a55fc6412101c3de894bcd480bca3e493eda47d1L171) `redeemTransaction` is called, which results in a lock acquire. We want to mock this in the test environment unless we wat to test the semaphore itself - or thats how I understood @Mogge